src/main/resources/application.yml

```diff
@@ -41,13 +41,13 @@ spring:
         registration:
           elite2-api:
             provider: hmpps-auth
-            client-id: ${oauth.client.id}
-            client-secret: ${oauth.client.secret}
+            client-id: ${offender.case.notes.client.id}
+            client-secret: ${offender.case.notes.client.secret}
             authorization-grant-type: client_credentials
             scope: read
         provider:
           hmpps-auth:
-            token-uri: ${oauth.endpoint.url}/oauth/token
+            token-uri: ${oauth.api.base.url}/oauth/token
```

Comment:
so this was the bug with the client credentials - hadn't amended to be the values that come from the helm config, but left the config below